ui/oopsyraidsy/data/00-misc/general.ts

```diff
@@ -62,6 +69,7 @@ const partyMitAbilityIdToDuration: { [id: string]: number } = {
   // melee
   '41': 15, // mantra
   // ranged
+  // TODO: troubadour/tactician/shield samba should match for overwriting each other
```

Comment:
Not sure I'm groking what the to-do is -- possible to clarify?

Reply:
Is it maybe that troub/tact/samba don't stack? I can't recall if they do or don't, most comps only ever run one phys ranged.

Reply:
The physical ranged party mitigations are effectively a role action (shared, mutually exclusive effect) while being implemented as job-specific actions, so the overwrite checking currently doesn't fire if, say, a BRD overwrites a MCH.

Reply:
Ah okay, thanks for the reminder, and the TODO makes sense.  Just wasn't processing it for some reason.